IMS/IMS/Controllers/InventoryController.cs

```diff
@@ -13,11 +13,62 @@ public InventoryController(IInventoryRepository InventoryRepository)
         {
             _InventoryRepository = InventoryRepository;
         }
+
         [HttpGet]
-        public IActionResult GetAll()
+        public async Task<IActionResult> Get(int page = 1, int limit = 10)
+        {
+            var inventory = await _InventoryRepository.GetInventoryAsync(page,limit);
```

Comment:
Rename inventory repository reference to match .NET naming standards:

```suggestion
            var inventory = await _inventoryRepository.GetInventoryAsync(page,limit);
```